ibis/backends/tests/test_array.py

```diff
@@ -547,6 +542,9 @@ def test_array_filter(con, input, output):
 
 @builtin_array
 @pytest.mark.notimpl(["polars"], raises=com.OperationNotDefinedError)
+@pytest.mark.broken(
+    ["flink"], raises=Py4JJavaError, reason="NullPointerException raised on execution"
+)
```

Comment:
Not sure what's happening here.

Reply:
This seems to be somehow related to the element type of the array: there are no issues executing `ARRAY_CONTAINS` queries with string element types, but an NPE when executing `ARRAY_CONTAINS` queries with `BIGINT` types.

Reply:
I added a second test case to `array_contains` for strings.